### PR TITLE
Add model

### PR DESCRIPTION
**Pull request by mdv5**
Added the Create a customer service and related test cases. Also added exception handlers in the routes.py file and cleaned up some comments in the models.py file.

### CODE REVIEW

File: tests/test_service.py [hunk 1]
```diff
@@ -10,32 +10,38 @@
 from unittest import TestCase
```
Comment (ddaisudov) [inline tests/test_service.py:10]: @mdv5 - why don't you rename this test_service file to test_routes.py to make it consistent with our change back to routes?

**Review by ddaisudov — COMMENTED** [review]
I added a comment about the test_service naming convention

File: tests/test_service.py (hunk 1 above)
Comment (mdv5) [reply tests/test_service.py:10]: Changed the name and committed.

Comment (ddaisudov) [reply tests/test_service.py:10]: amazing thanks so much! will review and merge

**Comment by ddaisudov** [comment]
Ran nosetests, and all 6 tests passed (see below):

Test Cases for Customer Model
- Create a customer and add it to the database
- Create a customer and assert that it exists

REST API Server Tests
- Create a new Customer
- Test index call

REST API Server Tests
- Create a new Customer
- Test index call

**Pull request merged**